app/src/main/java/com/dankideacentral/dic/authentication/OAuthWebViewFragment.java

```diff
@@ -28,10 +27,16 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
             @Override
             public boolean shouldOverrideUrlLoading(WebView view, String url) {
                 if (url.contains("oauth_verifier=")) {
-                    Intent intent = new Intent(getActivity().getApplicationContext(), TweetFeedActivity.class);
-                    intent.setData(Uri.parse(url));
+                    // Create user session
+                    TwitterSession.getInstance().createSession(Uri.parse(url),
```

Comment:
getInstance() is a static method so inline calls don't require an import. Sorry - just realized my previous comment didn't clear this up at all